packages/stack-shared/src/interface/crud/users.ts

```diff
@@ -27,6 +36,18 @@ export const usersCrudServerReadSchema = fieldSchema.yupObject({
   profile_image_url: fieldSchema.profileImageUrlSchema.nullable().defined(),
   signed_up_at_millis: fieldSchema.signedUpAtMillisSchema.required(),
   has_password: fieldSchema.yupBoolean().required().meta({ openapiField: { description: 'Whether the user has a password associated with their account', exampleValue: true } }),
+  client_metadata: fieldSchema.userClientMetadataSchema,
+  client_read_only_metadata: fieldSchema.userClientReadOnlyMetadataSchema,
+  server_metadata: fieldSchema.userServerMetadataSchema,
+  last_active_at_millis: fieldSchema.userLastActiveAtMillisSchema.required(),
+  contact_channels: fieldSchema.yupArray(contactChannelSchema).required(),
```

Comment:
do we want this on here? I'm kinda split, if we have this, why don't we have auth_methods? contact_channels also has its own endpoint, so there's probably a world where we don't wanna duplicate the information on the users endpoint

(GraphQL would answer this for us)